src/app/login/page.tsx

```diff
@@ -0,0 +1,88 @@
+'use client'
+
+import { createClientComponentClient } from '@supabase/auth-helpers-nextjs'
+import { useRouter } from 'next/navigation'
+import { useState } from 'react'
+
+import type { Database } from '@/app/auth/lib/database.types'
+import Input from "@/UI/Input";
+import Button from "@/UI/Button";
+
+export default function Login() {
+  const [isLogin,setIsLogin] = useState<boolean>(false)
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const router = useRouter()
+  const supabase = createClientComponentClient<Database>()
+
+  const handleSignUp = async () => {
+    await supabase.auth.signUp({
```

Comment:
Your components shouldn't have async functions in it, you need to avoid doing this in components